src/App.js

```diff
@@ -13,6 +13,7 @@ const triadQualities = {
 };
 const legalClefs = ['treble', 'bass', 'alto', 'tenor'];
 const legalTriads = ['M', 'm', 'aug', 'dim', 'sus2', 'sus4'];
+const legalInversions = ['inv0', 'inv1', 'inv2'];
```

Comment:
The param names are mostly-arbitrary. In this case, I did use string replace to fetch the number at the end, but there are many ways to achieve the same thing.